target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -183,6 +206,13 @@ define Device/tplink_eap623od-hd-v1
 endef
 TARGET_DEVICES += tplink_eap623od-hd-v1
 
+define Device/tplink_eap625-outdoor
+	$(Device/tplink_eap6XX-outdoor-base)
+	DEVICE_MODEL := EAP625-Outdoor
```

Comment:
Please at least add `HD` suffix to the full name here. You can use whatever short name you like as long as it's not ambiguous. But please make full name accurate and matches official name.